CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Changelog
 ### Improvements and Changes
 
 -   Type hints have been added to the `PauliTerm` class (@rht, gh-1075).
+-   'QAM.write_memory' now accepts either a Sequence of values or a single value. (@tommy-moffat, gh-658)
```

Comment:
```suggestion
-   `QAM.write_memory` now accepts either a Sequence of values or a single value. (@tommy-moffat, gh-658)
```

these are supposed to be backticks to get markdown "code" formatting, but I don't have the heart to "request changes" again!